config/initializers/new_framework_defaults_5_1.rb

```diff
@@ -9,6 +9,10 @@
 # Make `form_with` generate non-remote forms.
 Rails.application.config.action_view.form_with_generates_remote_forms = false
 
+# The belongs_to associations are required by default in Rails 5
+# we cannot opt into this right now, but it would be a goal for the future
+Rails.application.config.active_record.belongs_to_required_by_default = false
```

Comment:
Does this goal involve more than adding `inverse of` everywhere? (checking if it is a beginner-friendly issue)

Reply:
Hm, to be honest, I don't know 🙈 
I would guess it's probably going over all associations, adjusting / removing the validations, updating specs and factories, etc.

Reply:
Let's NOT make it a beginners issue ;-)

Reply:
:thinking: I'm not familiar with this inverse_of. I remember that at one point I was starting to use it, only to learn that Rails core was thinking about removing it again. Never followed up on that one. It would seem it's still there and heaved upon the `belongs_to` macro at that?

Reply:
🙈 thanx a million for pointing this out @carpodaster 🙏 
I since forever "just used"™️ `inverse_of`, because I thought it would do some behind the scenes optimization for looking up the association (see the beginning of [this blogpost from back in the days](https://www.viget.com/articles/exploring-the-inverse-of-option-on-rails-model-associations)).
**BUT** as it turns out, this was already added as a default behavior to Rails 4 (and therefore probably also Rails 5). It's probably still around for associations where the inverse cannot automatically be determined by the name - but for basic cases like most of the associations in the Teams App, this is obsolete 💡 

So I guess I can ditch the `inverse_of`s I added in this PR 🎉 

Learned something - yay!

Reply:
If I understand it correctly, the 'inverse of"  is (now?) required when validating associated records and/or on polymorphic associations.  https://github.com/rails/rails/pull/18937
I can't see if this is relevant for this PR.  If it isn't, please ignore this :-)